ddtrace/contrib/psycopg/__init__.py

```diff
@@ -60,23 +60,18 @@
     cursor = db.cursor()
     cursor.execute("select * from users where id = 1")
 """
-from ddtrace.internal.utils.importlib import require_modules
+# Required to allow users to import from `ddtrace.contrib.psycopg.patch` directly
```

Comment:
So the cycle here is that we try to import `psycopg` when bootstrapping and instrumenting all integrations, and then within this `psycopg/__init__.py` file we attempt to import `require_modules`? Wouldn't this mean that this problem isn't exclusive to the `psycopg` integration, but any of the integrations that use `require_modules`?

Reply:
Yes it seems like it but I am not sure why we don't see this issue in other integrations. We have been using require modules for almost a year. 

cc: @P403n1x87